x-pack/legacy/plugins/apm/public/components/app/ServiceMap/Popover/ServiceMetricList.tsx

```diff
@@ -0,0 +1,179 @@
+  defaultMessage: 'N/A'
+});
+
+interface MetricListProps {
+  serviceName: string;
+}
+
+export function ServiceMetricList({ serviceName }: MetricListProps) {
+  const {
+    urlParams: { start, end, environment }
+  } = useUrlParams();
+
+  // FIXME: Don't fetch if it's not a service. Perhaps this fetching should go
+  // in ServiceMetricList instead.
+  const { data = {} as ServiceNodeMetrics, status } = useFetcher(
+    callApmApi => {
+      if (serviceName && start && end) {
+        return callApmApi({
+          pathname: '/api/apm/service-map/service/{serviceName}',
+          params: {
+            path: {
+              serviceName
+            },
+            query: {
+              start,
+              end,
+              environment
+            }
+          }
+        });
+      }
+    },
+    [serviceName, start, end, environment],
+    {
+      preservePreviousData: false
+    }
+  );
+
+  const {
+    avgTransactionDuration,
+    avgRequestsPerMinute,
+    avgErrorsPerMinute,
+    avgCpuUsage,
+    avgMemoryUsage,
+    numInstances
+  } = data;
+  const isLoading = status === 'loading';
+
+  const listItems = [
+    {
+      title: i18n.translate(
+        'xpack.apm.serviceMap.avgTransDurationPopoverMetric',
+        {
+          defaultMessage: 'Trans. duration (avg.)'
+        }
+      ),
+      description: isNumber(avgTransactionDuration)
+        ? asDuration(toMicroseconds(avgTransactionDuration, 'milliseconds'))
+        : na
+    },
+    {
+      title: i18n.translate(
+        'xpack.apm.serviceMap.avgReqPerMinutePopoverMetric',
+        {
+          defaultMessage: 'Req. per minute (avg.)'
+        }
+      ),
+      description: isNumber(avgRequestsPerMinute)
+        ? `${avgRequestsPerMinute.toFixed(2)} ${tpmUnit('request')}`
+        : na
+    },
+    {
+      title: i18n.translate(
+        'xpack.apm.serviceMap.avgErrorsPerMinutePopoverMetric',
+        {
+          defaultMessage: 'Errors per minute (avg.)'
+        }
+      ),
+      description: avgErrorsPerMinute?.toFixed(2) ?? na
+    },
+    {
+      title: i18n.translate('xpack.apm.serviceMap.avgCpuUsagePopoverMetric', {
+        defaultMessage: 'CPU usage (avg.)'
+      }),
+      description: isNumber(avgCpuUsage) ? asPercent(avgCpuUsage, 1) : na
+    },
+    {
+      title: i18n.translate(
+        'xpack.apm.serviceMap.avgMemoryUsagePopoverMetric',
+        {
+          defaultMessage: 'Memory usage (avg.)'
+        }
+      ),
+      description: isNumber(avgMemoryUsage) ? asPercent(avgMemoryUsage, 1) : na
+    }
+  ];
+  return isLoading ? (
+    <LoadingSpinner />
+  ) : (
+    <>
+      {numInstances && numInstances > 1 && (
+        <EuiFlexItem>
+          <div>
+            <EuiBadge iconType="apps" color="hollow">
+              {i18n.translate('xpack.apm.serviceMap.numInstancesMetric', {
+                values: { numInstances },
+                defaultMessage: '{numInstances} instances'
```

Comment:
should it handle singular case? `1 instance` as well?

Reply:
No because it only shows up if `numInstances > 1`.